pkg/overlay/cache.go

```diff
@@ -200,27 +266,35 @@ func (cache *Cache) Put(ctx context.Context, nodeID storj.NodeID, value pb.Node)
 
 // Delete will remove the node from the cache. Used when a node hard disconnects or fails
 // to pass a PING multiple times.
-func (cache *Cache) Delete(ctx context.Context, id storj.NodeID) error {
+func (cache *Cache) Delete(ctx context.Context, id storj.NodeID) (err error) {
 	if id.IsZero() {
 		return ErrEmptyNode
 	}
+	defer mon.Task()(&ctx)(&err)
```

Comment:
shouldn't this happen before the if statement?

Reply:
the reason I'm excluding the obvious error case is because it probably isn't very informative in the output... but I can also move it upwards

Reply:
it just seems odd to exclude it but 🤷🏻‍♀️doesn't seem like a big deal

Reply:
ok, made consistent.